x-pack/plugins/infra/public/pages/metrics/inventory_view/components/waffle/conditional_tooltip.tsx

```diff
@@ -50,55 +52,69 @@ export const ConditionalToolTip = withTheme(({ theme, node, nodeType, currentTim
       },
     },
   });
-  const { nodes } = useSnapshot({
-    filterQuery: query,
-    metrics: requestMetrics,
-    groupBy: [],
-    nodeType,
-    sourceId,
-    currentTime,
-    accountId: '',
-    region: '',
-  });
+  const { nodes, loading } = useSnapshot(
+    {
+      filterQuery: query,
+      metrics: requestMetrics,
+      groupBy: [],
+      nodeType,
+      sourceId,
+      currentTime: requestTs,
+      accountId: '',
+      region: '',
+      includeTimeseries: false,
+    },
+    {
+      abortable: true,
+    }
+  );
 
   const dataNode = first(nodes);
   const metrics = (dataNode && dataNode.metrics) || [];
   return (
     <div style={{ minWidth: 200 }} data-test-subj={`conditionalTooltipContent-${node.name}`}>
       <div
         style={{
-          borderBottom: `1px solid ${theme?.eui.euiColorMediumShade}`,
-          paddingBottom: theme?.eui.euiSizeXS,
-          marginBottom: theme?.eui.euiSizeXS,
+          borderBottom: `1px solid ${euiTheme.colors.mediumShade}`,
```

Comment:
I think you can replace it by:
but I'm not sure if it should be `thin` or `thick`
```suggestion
          borderBottom: euiTheme.border.thin,
```